checkin.py

```diff
@@ -1,5 +1,8 @@
+#!/usr/bin/env python3
+
 import os
 import datetime
+import pytz
```

Comment:
@sainas  Hey. I have another concern about it. Do we need to install pytz. That will be inconvenient.
Can you used some library out of the box, like ZoneInfo.

Reply:
Cool.